api/swagger.yaml

```diff
@@ -2451,6 +2451,10 @@ definitions:
       VIP:
         type: "string"
         x-omitempty: false
+        x-go-type:
```

Comment:
Note to self; need to create a tracking ticket to evaluate where we can add `omitempty`  (and explicit "name") for the JSON structs. Also in light of json v2, where we can be more strict.